bread/formatters.py

```diff
@@ -9,41 +9,36 @@
 from django.db import models
 from django.utils.functional import Promise
 from django.utils.html import format_html, format_html_join, linebreaks, mark_safe
-from django_countries.fields import CountryField
 
 import bread.settings as app_settings
 
 from . import layout
 
 
-def format_value(value, fieldtype=None):
+def format_value(value):
     """Renders a python value in a nice way in HTML. If a field-definition has an attribute "renderer" set, that function will be used to render the value"""
-    if hasattr(fieldtype, "renderer"):
-        return fieldtype.renderer(value)
-
     if isinstance(value, bool) or value is None:
         return CONSTANTS[value]
 
-    # make referencing fields iterable (normaly RelatedManagers)
     if isinstance(value, models.Manager):
         value = value.all()
 
-    # If there is a hint passed via fieldtype, use the accoring conversion function first (identity otherwise)
-    # This is mostly helpfull for string-based fields like URLS, emails etc.
-    value = MODELFIELD_FORMATING_HELPERS.get(fieldtype.__class__, lambda a: a)(value)
-
     if isinstance(value, bool) or value is None:
         return CONSTANTS[value]
     if isinstance(value, datetime.timedelta):
         return as_duration(value)
-    if isinstance(value, datetime.datetime):
-        return as_datetime(value)
     if isinstance(value, numbers.Number):
         return f"{float(value):f}".rstrip("0").rstrip(".")
     if isinstance(value, models.fields.files.FieldFile):
         return as_download(value)
     if isinstance(value, Iterable) and not isinstance(value, (str, bytes, Promise)):
         return as_list(value)
+    if isinstance(value, str):
+        if "\n" in value:
+            return as_text(value)
+        if is_email_simple(value):
+            return as_email(value)
+
```

Comment:
Since we make the formatter independent of the model field, we need to add a bit of output-sugar manually.